README.md

```diff
@@ -44,6 +44,7 @@ It doesn't require any modification to your application's code and let's you col
     ```ini
     extension=apm.so
     ```
+    Note: APM depends on JSON, so the apm.so extension must be loaded **after** JSON!
```

Comment:
It seems Debian uses the same patch http://anonscm.debian.org/cgit/pkg-php/php.git/tree/debian/patches/0034-php-5.4.0-dlopen.patch